pg-index-health/src/test/java/io/github/mfvanek/pg/checks/host/BtreeIndexesOnArrayColumnsCheckOnHostTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health
+ *
+ * This file is a part of "pg-index-health" - a Java library for
+ * analyzing and maintaining indexes health in PostgreSQL databases.
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+package io.github.mfvanek.pg.checks.host;
+
+import io.github.mfvanek.pg.common.maintenance.Diagnostic;
+import io.github.mfvanek.pg.model.PgContext;
+import io.github.mfvanek.pg.model.index.Index;
+import io.github.mfvanek.pg.support.DatabaseAwareTestBase;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static io.github.mfvanek.pg.support.AbstractCheckOnHostAssert.assertThat;
+
+class BtreeIndexesOnArrayColumnsCheckOnHostTest extends DatabaseAwareTestBase {
+
+    private final BtreeIndexesOnArrayColumnsCheckOnHost check = new BtreeIndexesOnArrayColumnsCheckOnHost(getPgConnection());
```

Comment:
I prefer to use here an interface `DatabaseCheckOnHost<IndexWithColumns> check = `

Reply:
Done